src/selection.ts

```diff
@@ -200,6 +198,25 @@ export interface IntervalSelectionConfig extends BaseSelectionConfig<'interval'>
   mark?: BrushConfig;
 }
 
+export interface RegionSelectionConfig extends BaseSelectionConfig<'region'> {
+  /**
+   * A region selection also adds a path mark to depict the
+   * shape of the region. The `mark` property can be used to customize the
+   * appearance of the mark.
+   *
+   * __See also:__ [`mark` examples](https://vega.github.io/vega-lite/docs/selection.html#mark) in the documentation.
+   */
+  mark?: BrushConfig;
+
+  /**
+   * An array of field names whose values must match for a data tuple to
+   * fall within the selection.
+   *
+   * __See also:__ The [projection with `encodings` and `fields` section](https://vega.github.io/vega-lite/docs/selection.html#project) in the documentation.
+   */
+  fields?: FieldName[];
```

Comment:
How should this property work?  Shouldn't the fields always match the x and y encodings?